scripts/in_container/prod/entrypoint_prod.sh

```diff
@@ -68,8 +68,10 @@ function run_nc() {
     # Even if this message might be harmless, it might hide the real reason for the problem
     # Which is the long time needed to start some services, seeing this message might be totally misleading
     # when you try to analyse the problem, that's why it's best to avoid it,
-    local host="${1}"
-    local port="${2}"
+    local host
```

Comment:
it was good as it was before I think. Shorter and safe. The split to declaration/assignment should only be done if there is a "command" executed on the right side because it then can produce unexpected results.